src/client/Microsoft.Identity.Client/Internal/RequestContext.cs

```diff
@@ -23,6 +23,10 @@ internal class RequestContext
 
         public CancellationToken UserCancellationToken { get; }
 
```

Comment:
Two members have been added apparently just for logging. What are the other options?

Reply:
Changed